app/objects/c_operation.py

```diff
@@ -10,40 +10,77 @@
 
 
 REDACTED = '**REDACTED**'
+DEFAULT_REDACTED_FIELDS = {
+    'host_group[]': [
+        'group',
+        'server',
+        'location',
+        'display_name',
+        'host',
+    ],
+    'steps{}steps[]': [
+        'description',
+        'name',
+        'output',
+    ],
+    'adversary': [
+        'name',
+        'description',
+    ],
+    'adversary.phases{}[]': [
+        'name',
+        'description',
+    ],
+    'facts[]': [
+        'unique',
+        'value',
+    ],
+    'skipped_abilities[]{}[]': [
+        'ability_name',
```

Comment:
I think this may be overthought for the time being. let's see how the report and redaction changes and update from there